runtime-extension-model/src/main/java/org/mule/runtime/config/internal/dsl/utils/DslConstants.java

```diff
@@ -0,0 +1,250 @@
+/*
+ * Copyright 2023 Salesforce, Inc. All rights reserved.
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.config.internal.dsl.utils;
+
+import static java.lang.String.format;
+
+/**
+ * Mule DSL constants.
+ *
+ * This is an internal utils class (internal to the container, not the module), not to be considered part of the API. Backwards
+ * compatibility not guaranteed.
+ *
+ * @since 4.7
+ */
+public interface DslConstants {
```

Comment:
remove duplications, make internal in mule-core with directed exports for removing duplicates